pkg/core/tekton/builder/pipeline.go

```diff
@@ -0,0 +1,131 @@
+package builder
+
+import (
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PipelineBuilder holds the tekton pipeline definition.
+type PipelineBuilder struct {
+	Pipeline v1beta1.Pipeline
+}
+
+// NewPipelineBuilder creates a Pipeline with default values.
+func NewPipelineBuilder(name, namespace string) *PipelineBuilder {
```

Comment:
Are the functions from this file actually called anywhere?

Reply:
Sorry, scratch this, github wasn't showing me the whole diff...

Reply:
yes: https://github.com/fuseml/fuseml-core/pull/28/files#diff-86325b96c63cd747dca353c9a6952f87c71b87cf12a18231587eadfddb166096R154